internal/verify/verify_test.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+package verify
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/edgelesssys/constellation/v2/internal/logger"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseCerts(t *testing.T) {
+	validCert := `-----BEGIN CERTIFICATE-----
```

Comment:
I moved all the values you commented on to a testdata package. But instead of embedding them I put the values into constants to prevent accidental overwrites. Let's talk in case you want to stick to our current approach.